expat/lib/siphash.h

```diff
@@ -101,6 +104,8 @@
 #include <stddef.h> /* size_t */
 #include <stdint.h> /* uint64_t uint32_t uint8_t */
 
+#include <expat_config.h>
```

Comment:
`siphash.h` is autonomous as of today (i.e. it can be copied and work in isolation out of the box), hence the dedicated change log. While it is autonomous, we cannot include `expat_config.h` here but would duplicate needed macros in here, instead. At least so far.

The copy in here should have a prefixed name to not collide with other macros, e.g. `SIPHASH_FALLTHROUGH` would work.

Reply:
Done (both).